src/main/kotlin/org/malachite/estella/people/api/UserController.kt

```diff
@@ -35,6 +74,14 @@ class UserController(@Autowired private val userService: UserService) {
         return ResponseEntity.created(URI("/api/users/" + saved.id)).build()
     }
 
+    @CrossOrigin
+    @GetMapping("/{userId}")
+    fun getUser(@PathVariable userId: Int): ResponseEntity<User> {
```

Comment:
To, według mnie, można zrobić trochę lepiej. Jestem na telefonie, więc nie wiem, czy wyjdzie jak chce, ale możesz spróbować coś takiego:
```kotlin
fun(...) =
    userService.getUser(userId).let {
      ResponseEntity(it, OK) 
} 
```

Reply:
Znaczy tak naprawdę ja tylko zmieniłem kolejność tego kodu ale mogę to poprawić XD

Reply:
Nawet dodam obsługę błędu XD

Reply:
W sensie wiem, że to nie Twoje śmieci, ale zostawiamy kod lepszym, niż go zastalismy XDDD nie no, a tak serio, to nie obrażaj się za to, komentarze są do kodu, nie do Ciebie XDD

Reply:
Jakbym się obraził to bym przestał odpisywać na pytania do PR'a XD

Reply:
> Nawet dodam obsługę błędu XD

Jednak nie XD